src/Illuminate/Support/Collection.php

```diff
@@ -194,9 +194,9 @@ public function median($key = null)
             return $values->get($middle);
         }
 
-        return (new static([
+        return $this->newInstance([
```

Comment:
Isn't `static::make(...)` already creating a new instance?
This could have been used as a replacement for `$this->newInstance(...)`

Reply:
Yes, it is. But we can't use `$this` inside a static function